globals.go

```diff
@@ -653,6 +653,86 @@ var (
 	}
 )
 
+// UserRequester represents the allowed user-facing requester types.
+type UserRequester string
+
+const (
+	// User-facing requester types. These are equivalent in meaning to the above
+	// requesters, but are more user-friendly. These should only be used for
+	// user-facing functionality such as YAML configuration and expansions and
+	// should be translated into the true internal requester types so they're
+	// actually usable.
+	PatchVersionUserRequester       UserRequester = "patch"
```

Comment:
Welp I wanted to panic about having these special requester names but it looks like I approved them initially for expansions so I blame myself 🙃

Reply:
Yeah, I was wondering if I could just use the actual requester values, but the existing docs already described the requesters like this. 🤷